app/code/BVMyBlog/Blog/view/frontend/templates/postTmpl.phtml

```diff
@@ -0,0 +1,38 @@
+<?php
+/** @var \BVMyBlog\Blog\Block\Post $block*/
+
+$id = $block->getBlogId();
+$data = $block->getPost();
+?>
+<div>
+    <?php $post_id = $data->getPostId();
+        $title= $data->getTitle();
+        $post_content = $data->getContent();
+        $image_path = $data->getImagePath();
+        $created_at = $data->getCreationTime(); ?>
+    <h4>Id:</h4><p><?= $block->escapeHtml($post_id) ?></p>
+    <h4>Title:</h4><p><?= $block->escapeHtml($title) ?></p>
+    <h4>Content:</h4><p><?= $block->escapeHtml($post_content) ?></p>
+    <h4>Created at:</h4><p><?= $block->escapeHtml($created_at) ?></p>
+    <img src="<?= $block->escapeUrl($image_path) ?>"/>
+    <br/>
+    <button class="button-back-to-product"
+            data-mage-init='{"redirectUrl": {"event": "click",
+            "url": "<?= $block->escapeUrl($block->getBackUrl()) ?>"}}'
+            type="button"
+            title="Back to product"
+            id="backToProduct">
+        <span>Back to product</span>
+    </button>
+</div>
+
+<script>
+    require([
+        'jquery',
+        'redirectUrl'
+    ], function ($) {
+        'use strict';
+
+        $("#backToProduct").redirectUrl({url: "<?= $block->escapeUrl($block->getBackUrl()) ?>"});
+    });
+</script>
```

Comment:
Do we need this? **redirectUrl** widget should be inited by **data-mage-init** attribute